src/api/spec/features/webui/packages_spec.rb

```diff
@@ -175,4 +175,20 @@
       expect(page.source).to have_text('[1] this is my dummy logfile')
     end
   end
+
+  describe "with 'more info' link in flash error" do
+    let!(:broken_package) { create(:package_with_broken_service, name: "package_with_error", project: user.home_project) }
+    before do
+      login user
+      visit package_show_path(package: broken_package, project: user.home_project)
+    end
+
+    scenario 'show more info' do
+      click_link('more info')
+      expect(find('#flash-messages').text).not_to be_empty
+      expect(page).to have_css(".more_info")
+      expect(find('.more_info').text).not_to be_empty
+      expect(find(".btn-more").text).to eq('less info')
```

Comment:
I think it's worth to test that we actually show the content of the error / expanded error